Libplanet/Store/DefaultStore.cs

```diff
@@ -551,7 +571,6 @@ HashDigest<SHA256> blockHash
             HashDigest<SHA256> blockHash,
             IImmutableDictionary<Address, IValue> states)
         {
-            _statesCache.AddOrUpdate(blockHash, states);
```

Comment:
It seems caching after calling `GetBlockStates()` at least once, is it intended?
If then, I wonder why `SetBlockStates()` doesn't set cache. 🤔

Reply:
Oh sorry, it was moved to line 603. It was my mistake.